atomisticparsers/gromacs/parser.py

```diff
@@ -712,12 +713,12 @@ def get_composition(children_names):
             time_step = self.log_parser.get('input_parameters', {}).get('dt', 1.0) * ureg.ps
             if sec_system.time is not None:
                 self._system_time_map[round(ureg.convert(
-                    sec_system.time.magnitude, sec_system.time.units, ureg.picosecond), 5)] = n
+                    sec_system.time.magnitude, sec_system.time.units, ureg.picosecond), 5)] = system_counter
```

Comment:
I think we do not need system_counter, simply len(system_step_map).

Reply:
please include this in next pr

Reply:
oh sorry, I misunderstood, I thought you were saying nevermind below. I will change it now.

Reply:
It's ok its minor, can go with another pr.